### PR TITLE
Fix indenting lines 44-51

### PR DESCRIPTION
**Pull request by matAtWork**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR         | 
| Any Dependency Changes?  |
| License                  | MIT

### CODE REVIEW

**Comment by matAtWork** [comment]
This pull addresses https://github.com/babel/babel/pull/7076#discussion_r159251696

File: packages/babel-plugin-transform-async-to-promises/src/index.js [hunk 1]
```diff
@@ -42,11 +42,12 @@ function transformAsyncToPromises(api, options) {
         }; 
 
   if (options.codeGenerationOptions) {
-      let keys = Object.keys(options.codeGenerationOptions);
-      for (let i=0; i<keys.length;i++) {
-	  if (keys[i] in opts)
+    let keys = Object.keys(options.codeGenerationOptions);
+    for (let i=0; i<keys.length;i++) {
+	    if (keys[i] in opts) {
```
Comment (ljharb) [inline packages/babel-plugin-transform-async-to-promises/src/index.js]: this still needs one more space, so that it's two spaces indented from line 46

Comment (matAtWork) [reply packages/babel-plugin-transform-async-to-promises/src/index.js]: I can't see it in the online editor (it looks like there are two spaces) - can you fix it?

Comment (ljharb) [reply packages/babel-plugin-transform-async-to-promises/src/index.js]: I can't; this is a PR to your fork (from your fork), and I don't have write access there.

**Review by ljharb — APPROVED** [review]
LGTM

**Pull request merged**